ytypes/schema_tests/posix_regex_benchmark_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Google Inc.
```

Comment:
(nit) Is this too specific of a test file name? Should we just put this in the existing test files?

Reply:
My thought is it makes it clear where the benchmark tests are, because the other benchmark files are currently also very specific:

diff_benchmark_test.go
leafref_benchmark_test.go